### PR TITLE
fix(EMS-618): Heading href link

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issue: 
* Heading href didn't link to the right page.
* On insurance pages, was going to quote start
* All pages did not have functioning href on heading

# Changes made:
* Added to core and single page variables so href is on every page
* Linked href to insurance start for insurance pages
* Updated unit tests
* Added to e2e tests to test href

### CODE REVIEW

File: src/ui/types/page-variables.d.ts [hunk 1]
```diff
@@ -37,6 +39,7 @@ interface SingleInputPageVariablesInitialInput {
 interface SingleInputPageVariablesInput extends SingleInputPageVariablesInitialInput {
   FIELD_ID: string;
   PRODUCT: PageVariablesProduct;
+  START_ROUTE?: string;
```
Comment (ttbarnes) [inline src/ui/types/page-variables.d.ts:42]: Should `START_ROUTE` be required?

Comment (Zainzzkk) [reply src/ui/types/page-variables.d.ts:42]: fixed

**Review by abhi-markan — CHANGES_REQUESTED** [review]
All looks good, apart from comments mentioned by @ttbarnes

File: src/ui/server/middleware/required-data-provided/quote/index.ts [hunk 2]
```diff
@@ -103,7 +104,7 @@ export const requiredQuoteEligibilityDataProvided = (req: Request, res: Response
   const routesArray = getRoutesAsArray(ROUTES.QUOTE);
 
   // array of routes that do not require any data checks.
-  const irrelevantRoutes = [BUYER_COUNTRY, CANNOT_APPLY, GET_A_QUOTE_BY_EMAIL, NEED_TO_START_AGAIN];
+  const irrelevantRoutes = [BUYER_COUNTRY, CANNOT_APPLY, GET_A_QUOTE_BY_EMAIL, NEED_TO_START_AGAIN, START];
```
[the remark this answers is not in the document]
Comment (Zainzzkk) [reply src/ui/server/middleware/required-data-provided/quote/index.ts:107]: done - no e2e test needed

**Review by ttbarnes — APPROVED** [review]

**Review by ttbarnes — APPROVED** [review]

**Review by abhi-markan — APPROVED** [review]

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Failing test, please address.

**Pull request merged**